build/pkgs/vendoring/dependencies

```diff
@@ -0,0 +1,4 @@
+$(PYTHON) rich click toml | $(PYTHON_TOOLCHAIN)
```

Comment:
There is no `toml` package in Sage. Should this be `tomli` instead?

Reply:
the toml package was removed from Sage in the meantime; we'll have to re-add it for sagemath-mpmath